src/refg.cpp

```diff
@@ -68,9 +69,52 @@ refg_t refg_registry_t::as_refg(std::string_view config) const
 	return ref;
 }
 
+std::string_view refg_registry_t::_try_refg_as_sv_from_file(refg_t ref) const
+{
+	std::string path{fmt::format("{}.hash", ref)};
+	if (!std::filesystem::exists(path)) {
+		path = resolve_datafile_path(prepend_dir(data_dir(), path));
+	}
+	if (!std::filesystem::exists(path)) {
+		return {};
+	}
+	std::FILE* file = std::fopen(path.c_str(), "rb");
+	if (!file) {
+		println("Could not open file '{}'", path);
```

Comment:
should print the strerr as well. although I think maybe just using the line reader would be less reinvention

Reply:
Using line_reader